### DISCUSSION TITLE
How to change backbone network of RetinaNet

### DISCUSSION THREAD

**Discussion by ibrahim014** [open-mmlab/mmdetection#9724] [General] [answered]
Hi,

I want to change the RetinaNet backbone from ResNet50 to VGG16 or ResNet101. Can you please let me know how can we change it in config file?

**Comment by ZwwWayne** [comment, marked as answer]
See examples in https://github.com/open-mmlab/mmdetection/blob/master/configs/retinanet/retinanet_r101_fpn_1x_coco.py and  https://github.com/open-mmlab/mmdetection/tree/master/configs/retinanet
